packages/react-positioning/src/createArrowStyles.ts

```diff
@@ -0,0 +1,72 @@
+import type { MakeStylesStyleRule } from '@fluentui/make-styles';
+import type { Theme } from '@fluentui/react-theme';
+
+/**
+ * Helper that creates a makeStyles rule for an arrow element.
+ * For runtime arrow size toggling simply create extra classnames to apply to the arrow element
+ *
+ * ```ts
+ *   makeStyles({
+ *     arrow: createArrowStyles(),
+ *     smallArrow: { aspectRatio: 1, width: '2px' }
+ *   })
+ *   ...
+ *
+ *   state.arrow.clasName = mergeClasses(styles.arrow, styles.smallArrow)
+ * ```
```

Comment:
```suggestion
 * ```ts
 *   makeStyles({
 *     arrow: createArrowStyles(),
 *     smallArrow: createArrowStyles({ aspectRatio: 1, width: '2px' })
 *   })
 *   ...
 *
 *   state.arrow.className = styles.arrow
 * ```
```

I see that `createArrowStyles()` has `size` param. Should be usage corrected?

Reply:
I can include both variations.... because the size param will not handle dynamic runtime changes

Reply:
Can you please clarify what means "runtime"? Styles should be static in general otherwise build time transform will not handle them properly

Reply:
Updated the description in [ef12cda](https://github.com/microsoft/fluentui/pull/20541/commits/ef12cda12b7046235f4550ddf7e3cc9e028d4632)